static/css/devhub/new-landing/common.less

```diff
@@ -71,25 +71,85 @@
 }
 
 .DevHub-content-header {
-  font-size: 40px;
   max-width: 66%;
 }
 
 .DevHub-content-image-wrapper {
   max-width: 33%;
-  padding: 30px 0 30px 30px;
+  padding: 30px 0;
   position: absolute;
-  right: 20px;
   top: 20px;
+  right: 20px;
   z-index: 2;
 }
 
+.DevHub-content-image-wrapper--publish-addon {
+  right: 0;
+}
 
 .DevHub-content-copy {
   position: relative;
   z-index: 4;
 }
 
+.DevHub-callout-box {
+  border: 8px solid #FDC900;
+  background: #fff;
+  padding: 0 20px 20px;
+  // Override default padding on container
+  // with negative margin.
+  margin: 0 -20px -20px;
+
+  @media @medium {
+    box-shadow: 8px 8px 0 0 rgba(0,0,0,0.15);
+    margin: -50px auto 0 auto;
+    max-width: 75%;
+  }
+}
+
+.DevHub-callout-header {
+  font-size: 22px;
+  background: url(../../../img/developers/new-landing/callout-header-fox.svg) no-repeat 100% 50%;
+  padding-right: 35px;
+  min-height: 50px;
+}
+
+.DevHub-content-header--manage {
+  max-width: 100%;
+  text-align: center;
+}
+
+.DevHub-content-container--manage {
+  flex-direction: column;
+  background-image: url(../../../img/developers/new-landing/manage-addon.png);
```

Comment:
Yeah a lot of CSS syntax grammar seems to trail, I have similar issues with the newer stuff.